_build/detail.hbs

```diff
@@ -26,15 +26,20 @@
       <h4>Additional info</h4>
       <p>{{{mdExternal AdditionalInfoExternal}}}</p>
       {{/if}}    
- 
-      {{#if EOBrowser}}
-      <h4>Explore</h4>
-      <p><a href="{{EOBrowser}}" target="_blank"><button class="label label-info" > Open in EO Browser </button></a> </p>
-      {{/if}}  
+
       {{#if EDCBrowser}}
       <h4>Explore</h4>
-      <p><a href="{{EDCBrowser}}" target="_blank"><button class="label label-info" > Open in EDC Browser </button></a> </p>
-      {{/if}} 
+      <p><a href="{{EDCBrowser}}" target="_blank"><button class="label label-info"> Open in EDC Browser </button></a>
+            {{#if EOBrowser}}
+            <a href="{{EOBrowser}}" target="_blank"><button class="label label-info" style="margin:10px"> Open in EO Browser </button></a> </p>
+            {{/if}} 
+      {{else}}
+            {{#if EOBrowser}}
+            <h4>Explore</h4>
+            <p><a href="{{EOBrowser}}" target="_blank"><button class="label label-info" > Open in EO Browser </button></a> </p>
+      {{/if}}
```

Comment:
That's a nice solution since handlebars doesn't have a way to handle `or` logical operator.
I checked with both orders (first EDCBrowser link and then EOB and vice versa) and includes both of the links in both cases.

I would just indent this `{{/if}}` to the same level as the other 3 lines for this `if` part so it's easier to see what belongs together for the people changing the code.

Reply:
Thanks for catching this! Indent is added.

Reply:
Thanks, looks good.